test/grouping.jl

```diff
@@ -1,6 +1,7 @@
 module TestGrouping
 
-using Test, DataFrames, Random, Statistics, PooledArrays, CategoricalArrays, DataAPI
+using Test, DataFrames, Random, Statistics, PooledArrays, CategoricalArrays, DataAPI,
```

Comment:
I would not make a line break here :)

Reply:
Well without it the line takes 94 chars... :-)

Reply:
as you like then :)